apps/src/templates/sectionProgress/progressTables/ProgressTableLevelBubble.jsx

```diff
@@ -35,127 +46,124 @@ class ProgressTableLevelBubble extends React.PureComponent {
     bubbleSize: BubbleSize.full
   };
 
-  renderContent() {
+  constructor(props) {
+    super(props);
+    this.createBubbleElement = this.createBubbleElement.bind(this);
+  }
+
+  render() {
+    const bubble = (
+      <CachedElement
+        elementType={'BasicBubble'}
+        cacheKey={this.getCacheKey()}
+        createElement={this.createBubbleElement}
```

Comment:
since the children get passed into the parent as a prop, i was assuming that doing it that way would result in a call to `createElement` for the child even when we have it in the cache, which is what this component is intended to avoid. i'm sure that digging in to confirm that behavior would result in some valuable learnings, but i'm going to stick with this for now in the interest of time.